datalad/support/tests/test_annexrepo.py

```diff
@@ -1134,7 +1134,8 @@ def test_annex_copy_to(origin, clone):
     repo.get("test-annex.dat")
     # now it has:
     eq_(repo.copy_to("test-annex.dat", "target"), ["test-annex.dat"])
-    eq_(repo.copy_to(["INFO.txt", "test-annex.dat"], "target"), ["test-annex.dat"])
+    # and will not be copied again since it was already copied
+    eq_(repo.copy_to(["INFO.txt", "test-annex.dat"], "target"), [])
```

Comment:
This is problematic, I think. You changed to use `_run_annex_command_json` within `AnnexRepo.copy_to`. But this one tends to swallow failures and to just return empty list instead. This might be fine in general but in combination with that change to the test, we don't test whether it works with a list passed into it. It could always fail on lists now and we wouldn't notice.